run/helloworld/main.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START run_helloworld_server]
+
+// Sample run-helloworld is a minimal Cloud Run service.
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+)
+
+func main() {
+	log.Print("helloworld: starting server...")
```

Comment:
Use `log.SetPrefix` rather than manually prefixing with `helloworld:`?

Reply:
While I like having the prefix, it's not worth adding another line of code or teach a non-idiomatic approach. Removing.